core/market/src/protocol/discovery.rs

```diff
@@ -201,13 +204,15 @@ impl Discovery {
 
         log::debug!("Broadcasting unsubscribes. count={}", offer_ids.len());
         let bcast_msg = SendBroadcastMessage::new(UnsubscribedOffersBcast { offer_ids });
+        counter!("market.offers.unsubscribes.broadcasts.net", 1);
```

Comment:
I'd also report a histogram (`value!()`) for number of unsubscribes bcasted (`offer_ids.len()`)
```suggestion
        counter!("market.offers.unsubscribes.broadcasts.net", 1);
        value!("market.offers.unsubscribes.broadcasts.size", offer_ids.len());
```